pkg/machines/components/storagePools/storagePoolDelete.jsx

```diff
@@ -86,33 +131,83 @@ export class StoragePoolDelete extends React.Component {
     }
 
     render() {
-        const { storagePool } = this.props;
+        const { storagePool, vms } = this.props;
         const id = storagePoolId(storagePool.name, storagePool.connectionName);
         const volumes = storagePool.volumes || [];
 
+        const usage = getStorageVolumesUsage(vms, storagePool);
+        let vmsUsage = [];
+        for (let property in usage)
+            vmsUsage = vmsUsage.concat(usage[property]);
+
+        vmsUsage = vmsUsage.join(', ');
+        const showWarning = () => {
+            if (canDeleteOnlyWithoutVolumes(storagePool, vms) && this.state.deleteVolumes) {
+                return (
+                    <span id={`delete-${id}-idle-message`}>
+                        <i className='pficon pficon-info' />
+                        {_("Pool's volumes are used by: ") + vmsUsage + '. '}
```

Comment:
It's unclear this is talking about VMs.
I would change the string to something like:
_Pool is used by the VMs test1, test2, test3.
All disks need to be detached before you can delete the pool._

Reply:
And maybe bold the names of the VMs.

Reply:
Done